### DISCUSSION TITLE
Cause: [HystrixRuntimeException]{soa.xxxtimed-out and no fallback available.} [TimeoutException]{null} " }

### DISCUSSION THREAD

**Discussion by Cuieney** [midwayjs/midway#3366] [Q&A]
### 🐛 Describe the bug（描述 bug）

{
"Message": "TargetUrl: [http://xxx.xxx.com:80/monitor/saveRoomSnapshotInfo](http://xxx.xxx.com/monitor/saveRoomSnapshotInfo)
Cause: [HystrixRuntimeException]{soa.guidance timed-out and no fallback available.} [TimeoutException]{null} "
}

### Midway Versions（Midway 版本）

Cause: [HystrixRuntimeException]{soa.xxxx timed-out and no fallback available.} [TimeoutException]{null} "
}

post接口多次并发请求出现这个问题

midway: 3.0版本

**Comment by czy88840616** [comment]
需要提供详细信息，是其他服务请求 midway 服务？需要业务是否有报错，堆栈，描述，代码如何返回，中间件，拦截器的情况。

**Reply by Cuieney** [reply]
我这边在确认下，midway控制台还没看到具体的错误，应该是还没到midway服务。具体的流程是：h5页面请求接口（先到h5网关，网关请求具体的接口服务，在h5页面接口高并发的时就会出现网关层的报错，无法连接我的这个node（midway）服务（timeout））
